pegen/pegen.h

```diff
@@ -7,6 +7,12 @@
 #include <Python-ast.h>
 #include <pyarena.h>
 
+enum INPUT_MODE {
+    FILE_INPUT,
+    STRING_INPUT,
+    FSTRING_INPUT
```

Comment:
Do we need to distinguish `FSTRING_INPUT` from `STRING_INPUT`? The filename string will be different but the behaviour is the same, no?

Reply:
Right, I just thought that this could be somehow useful in the future. Should I remove it?

Reply:
> Right, I just thought that this could be somehow useful in the future. Should I remove it?

As you prefer, I would say yes just to make it a bit more clear that there are only two modes right now to understand.

Reply:
You know best.